packages/babel-cli/src/babel/dir.js

```diff
@@ -61,7 +61,9 @@ module.exports = function (commander, filenames) {
     }
   }
 
```

Comment:
Babel is generally pretty strict about arguments. It would be good to show an error if `commander.skipInitialBuild` is true but `commander.watch` is false.